crates/bevy_remote/src/schemas/open_rpc.rs

```diff
@@ -0,0 +1,100 @@
+//! Module with trimmed down `OpenRPC` document structs.
+//! It tries to follow this standard: <https://spec.open-rpc.org>
+use bevy_platform_support::collections::HashMap;
+use bevy_utils::default;
+use serde::{Deserialize, Serialize};
+
+use crate::RemoteMethods;
+
+/// Represents an `OpenRPC` document as defined by the `OpenRPC` specification.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OpenRpcDocument {
+    /// The version of the `OpenRPC` specification being used.
+    pub openrpc: String,
+    /// Informational metadata about the document.
+    pub info: InfoObject,
+    /// List of RPC methods defined in the document.
+    pub methods: Vec<MethodObject>,
+    /// Optional list of server objects that provide the API endpoint details.
+    pub servers: Option<Vec<ServerObject>>,
+}
+
+/// Contains metadata information about the `OpenRPC` document.
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct InfoObject {
+    /// The title of the API or document.
+    pub title: String,
+    /// The version of the API.
+    pub version: String,
+    /// An optional description providing additional details about the API.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub description: Option<String>,
+    /// A collection of custom extension fields.
+    #[serde(flatten)]
+    pub extensions: HashMap<String, serde_json::Value>,
+}
+
+impl Default for InfoObject {
+    fn default() -> Self {
+        Self {
+            title: "Bevy OpenRPC".to_owned(),
```

Comment:
```suggestion
            title: "Bevy Remote Protocol".to_owned(),
```
I think this makes more sense as a title for the API itself.